pkg/auth/providers/saml/saml_client.go

```diff
@@ -276,7 +276,7 @@ func (s *Provider) HandleSamlAssertion(w http.ResponseWriter, r *http.Request, a
 	userPrincipal, groupPrincipals, err = s.getSamlPrincipals(config, samlData)
 	if err != nil {
 		log.Error(err)
-		http.Redirect(w, r, redirectURL+"errorCode=422&errorMsg=Invalid saml attributes", http.StatusFound)
+		http.Redirect(w, r, redirectURL+"errorCode=422&errorMsg=invalidSamlAttrs", http.StatusFound)
```

Comment:
@mrajashree can you see other places using the same error message - `invalidSamlAttrs`? If yes, do you think defining it as a constant would make sense?

Reply:
@alena1108 I saw only one place returning errorMsg, so only changed it here